spec/next_rails/bundle_report_spec.rb

```diff
@@ -0,0 +1,30 @@
+require_relative ".././spec_helper"
+require_relative "../../lib/next_rails/bundle_report"
+
+RSpec.describe NextRails::BundleReport do
+  describe "#compatible_ruby_version" do
```

Comment:
We should stub the requests to rubygem's API during tests, this is doing real requests and can lead to flaky/slower specs if the rubygems servers are having issues.

Extracting the request to rubygems into some helper methods should make it easy to stub and hardcode some json responses.